### DISCUSSION TITLE
Implement in the core a logic similar to Yireo_EmailOverride

### DISCUSSION THREAD

**Discussion by boesbo** [OpenMage/magento-lts#2122] [Ideas]
### Description (*)
It would be useful to implement a logic similar to https://github.com/yireo-magento1/Yireo_EmailOverride in the core.

### Benefits

This would allow you to translate the email files directly into your custom template.

### Additional
If this functionality is in line with the business logic, I will develop it.

**Comment by simbus82** [comment]
And a great add is... Email Tester from Yireo!
https://github.com/yireo-magento1/Yireo_EmailTester

**Comment by Flyingmana** [comment]
biggest question for me is, why not using the existing module providing the functionality, instead of rebuilding it?

**Comment by boesbo** [comment]
> biggest question for me is, why not using the existing module providing the functionality, instead of rebuilding it?

Because the module now seems abandoned, if it is considered a function in line with the business logic, having it within the project would guarantee its maintenance.

**Comment by Flyingmana** [comment]
Sounds good for me, but with the size its maybe easier to fork it into our organisation, and include it via composer. (I have already a script prepared, which would generate a download for releases which includes it, for people who are not using composer)
At least temporary till we know its not causing any bigger issues.

Do you think thats ok?

**Comment by boesbo** [comment]
> Sounds good for me, but with the size its maybe easier to fork it into our organisation, and include it via composer. (I have already a script prepared, which would generate a download for releases which includes it, for people who are not using composer)
> At least temporary till we know its not causing any bigger issues.
> 
> Do you think thats ok?

I think it's perfect!

**Comment by Flyingmana** [comment]
got it confirmed to be abandoned: https://twitter.com/jissereitsma/status/1421441415440211969

Forked repository: https://github.com/OpenMage/Yireo_EmailTester

I suggest to change the vendor in composer.json from `yireo/magento1-emailtester` to `openmage/yireo-emailtester`, but leave the vendor namespace for the code untouched for now.

I added same Review Rules as for the LTS repository, as we plan to include it directly.
